plugins/plugin-chart-echarts/src/BoxPlot/types.ts

```diff
@@ -33,4 +33,9 @@ export type BoxPlotFormDataWhiskerOptions =
 
 export type BoxPlotFormXTickLayout = '45°' | '90°' | 'auto' | 'flat' | 'staggered';
 
+export interface EchartsBoxPlotChartProps extends ChartProps {
+  rawFormData: BoxPlotQueryFormData;
```

Comment:
`TableChart` uses `rawFormData` to reuse the strong typing of formData fields in `snake_case` (e.g. `rawFormData.row_limit`) , but it seems your `BoxPlot` is using `formData` and `camelCase` (`formData.rowLimit`). So you should probably either set `rawFormData` here to `formData` or use `rawFormData` in `transformProps`.

The whole thing is a little confusing, maybe we should just do a db migration to change all chart formData to use `camelCase`...

Reply:
done